backend/src/main/java/com/redhat/cloud/notifications/db/Query.java

```diff
@@ -10,6 +10,7 @@
 public class Query {
 
     private static final Pattern SORT_FIELD_PATTERN = Pattern.compile("^[a-z0-9._-]+$", CASE_INSENSITIVE);
+    private static final int DEFAULT_RESULTS_PER_PAGE  = 20;
```

Comment:
This should probably be mentioned in the OpenAPI doc for all affected endpoints.

Reply:
Added the `@DefaultParam` annotation, that updates the openapi.json